src/renderer/reducers/settings.js

```diff
@@ -275,7 +275,10 @@ const handlers: Object = {
     { payload }: { payload: { lastSeenDevice: DeviceModelInfo, latestFirmware: any } },
   ) => ({
     ...state,
-    lastSeenDevice: payload.lastSeenDevice,
+    lastSeenDevice: {
```

Comment:
I would prefere this one : 

Object.assign({}, state.lastSeenDevice, payload.lastSeenDevice)

If one element is undefined it will not affect the object you are building.
But it you prefere this approach then you can resolve.